Dacaspex/Exhibit
# #329 Added axis labels and tooltip to graph.

#### What have I done
Fixes #329.
Adds axis labels to the graph. X-axis simply gets a static time label. Y-axis needs to update when the selected GraphMetric changes. Also formatted the tooltip when hovering over a point in the graph.

#### Testing
* Import data.
* Open graph.
* Check axis labels, also after switching GraphMetric.
* Also hover over data points to see the tooltip.

#### Addresses SRD section(s)
Sections about GraphMetric.
